dicoogle/src/main/resources/webapp/js/external-requires.js

```diff
@@ -5,7 +5,8 @@
  */
 const EXTERNAL_MODULES = {
   react: require("react"),
-  "react-router": require("react-router"),
+  "react-router": require("react-router-dom"),
```

Comment:
Considering that this is a breaking change to any web plugin using react-router, I might as well drop the external module instead.

```suggestion
```